src/components/pokemon/Pokemon.jsx

```diff
@@ -0,0 +1,58 @@
+import './Pokemon.css'
+import {useEffect, useState} from "react";
+import axios from "axios";
+
+function Pokemon({endpoint}) {
+    const [pokemon, setPokemon] = useState([])
+    const [error, toggleError] = useState(false)
+    const [loading, toggleLoading] = useState(false)
+
+    useEffect(() => {
+
+
+        async function fetchData() {
+            toggleError(false)
+            toggleLoading(true)
+
+            try {
+                const response = await axios.get(endpoint)
+                setPokemon(response.data)
+            } catch (e) {
+                console.error('Fout bij ophalen')
+            }
+        }
+
+
+    }, []);
+
+    return (
+
+        <article className='pokemon-card'>
+            {Object.keys(pokemon).length > 0 &&
```

Comment:
Door `pokemon` default `undefined` te maken kan je check hier ook wat simpeler
```suggestion
            {pokemon &&
```